src/app/zwiazek/kalendarz/_components/Dates/Dates.tsx

```diff
@@ -0,0 +1,21 @@
+export const addMonth = (date: Date, offset = 1) =>
+  new Date(date.setMonth(date.getMonth() + offset));
+
+export const today = new Date();
+
+export const daysOfWeek = ["Pon", "Wt", "Śr", "Czw", "Pt", "Sob", "Ndz"];
+
+export const months = [
```

Comment:
Both `daysOfWeek` and `months` might not be needed. We can use `pl-PL` locale with proper options.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString#options